### ISSUE TITLE
FilteredSelectMultiple design is broken inside custom form

### ISSUE THREAD

**Issue by kwangsooshin** [otto-torino/django-baton#51]
Hi,

I have a problem when I use django-baton.

FilteredSelectMultiple design is broken when I use custom form.

Here is my custom form codes:
```python
class CustomCreationForm(forms.ModelForm):
    keys = forms.ModelMultipleChoiceField(
        queryset=Key.objects.all(),
        required=True,
        widget=FilteredSelectMultiple(
            verbose_name='key',
            is_stacked=False
        )
    )

    users = forms.ModelMultipleChoiceField(
        queryset=User.objects.all(),
        required=True,
        widget=FilteredSelectMultiple(
            verbose_name='user',
            is_stacked=False
        )
    )

    class Meta:
        model = UserKey
        fields = '__all__'
```

I use FilteredSelectMultiple and it is work at default django admin template.

But, when I use django-baton, it's broken such as:

![screencapture-0-0-0-0-6008-adminudi-userudikey-add-2020-05-26-14_23_37](https://user-images.githubusercontent.com/29035439/82863464-377dfd00-9f5d-11ea-8231-f38c428d4233.jpg)

Is it bug in baton or my code...?

**Comment by abidibo** [comment]
Hi @kwangsooshin ,
the problem is that the selector widget is not wrapped by the same container as it happens when using  `filter_horizontal` prop. By the way, I'll fix it now. Thanks!

**Comment by abidibo** [comment]
Fixed in django-baton==1.7.2
